test/index.test.ts

```diff
@@ -106,47 +106,48 @@ describe("mkdist", () => {
     expect(writtenFiles.sort()).toEqual(
       [
         "dist/README.md",
-        "dist/bar.d.ts",
+        "dist/bar.d.mts",
         "dist/bar.mjs",
         "dist/demo.css",
-        "dist/dir-export.d.ts",
+        "dist/dir-export.d.mts",
         "dist/dir-export.mjs",
         "dist/foo.mjs",
         "dist/foo.d.ts",
+        "dist/foo.d.mts",
```

Comment:
In this case there is an existing `foo.d.ts` in the source, but the generated `foo.d.mts` will be used instead. Would it be good to copy `foo.d.ts` and rename it? The developer could rename it to `foo.d.mts` in the `src` dir, but that would not work if they are generating both cjs and esm.